tests/src/test/scala/com/nvidia/spark/rapids/CastOpSuite.scala

```diff
@@ -333,8 +336,19 @@ class CastOpSuite extends GpuExpressionTestSuite {
     assert(unsupported == expected)
   }
 
+  private def getCastKey(ansiEnabled: Boolean): Class[Expression] = {
```

Comment:
this is same code as above can we make utility function?

Reply:
Sure, good suggestion